### ISSUE TITLE
Inggrit : Portfolio 

### ISSUE THREAD

**Issue by inggritenovaleta** [HackYourFutureBelgium/class-13-14#332]
- [ ] [repo](https://github.com/_/_) (with a complete README)
- [ ] [live demo](https://_.github.io/_)
- [/planning](https://github.com/_/_/tree/master/planning)
  - [ ] constraints
  - [ ] backlog
  - [ ] wireframe
  - [ ] development strategy
  - [ ] retrospective
- [ ] [project board](https://github.com/_/_/projects/1)

**Comment by inggritenovaleta** [comment]
I actually have a question. In the first months we started the HYF, we were making our webpage style using Figma, is this project going towards the same creation but more with Javascript?

**Comment by colevandersWands** [comment]
@inggritenovaleta , exactly.  You can update the design you made before to include a projects portfolio section, or start over from scratch if you like

**Comment by inggritenovaleta** [comment]
Alright, I have 50:50 of ideas from the last one and this new one. Thank you for the help
